app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+
+  def create
+    @user = User.new(permitted_params)
+    if @user.save
+      redirect_to users_path
+    else
+      flash[:errors] = @user.errors.full_messages
```

Comment:
What was the thought process behind wanting to handle the errors with flash messages?